keyboards/palette1202/readme.md

```diff
@@ -0,0 +1,24 @@
+# Palette1202
+
+---
+
+![Palette1202](https://palette1202.nilgiri-tea.net/images/palette1202.jpg)
+
+A left hand device with rotary encoder, for artists.
+
+Keyboard Maintainer: [niltea](https://github.com/niltea)  
+Hardware Supported: Palette1202
+Hardware Availability: [Pixiv Booth](https://booth.pm/)
+[PCB & Case](https://github.com/niltea/Palette1202)
+
+Make example for this keyboard (after setting up your build environment):
+
+- default
+
+		make palette1202:default:avrdude
```

Comment:
We have a universal flash target now. No need to specify a specific target

```suggestion
		make palette1202:default:flash
```

Reply:
I didn't know it, thanks!